src/vehicle.cpp

```diff
@@ -1394,12 +1394,7 @@ bool vehicle::fold_up() {
     }
 
     // create a folding [non]bicycle item
-    item bicycle;
-    if (can_be_folded) {
-        bicycle.make( "generic_folded_vehicle" );
-    } else {
-        bicycle.make( "folding_bicycle" );
-    }
+    item bicycle( can_be_folded ? "generic_folded_vehicle" : "folding_bicycle", calendar::turn );
```

Comment:
Can easily be resolved by changing to `convert` but it would be interesting to know why this is? I'll have a look but do you have any suggestions?